gcore/src/lib.rs

```diff
@@ -67,7 +67,7 @@ pub mod errors;
 pub mod exec;
 pub mod msg;
 pub mod prog;
-pub mod stack_buffer;
+pub extern crate gear_stack_buffer as stack_buffer;
```

Comment:
@breathx asked me to use `pub use gear_stack_buffer` here

Reply:
@ark0f whats the matter?

Reply:
Since Rust 2018 edition:
- extern crate is no longer needed in 99% of circumstances.
- An exception: "sysroot" crates, `#[macro_use] extern crate bar;`

Source: https://doc.rust-lang.org/edition-guide/rust-2018/path-changes.html